src/components/resources/MedicationDispense/MedicationDispense.test.js

```diff
@@ -32,17 +33,55 @@ describe('should render Device component properly', () => {
       fhirVersion: fhirVersions.STU3,
     };
 
-    const { container, getByTestId } = render(
+    const { container, getByTestId, queryAllByTestId } = render(
       <MedicationDispense {...defaultProps} />,
     );
     expect(container).not.toBeNull();
 
     expect(getByTestId('title').textContent).toContain('Capecitabine');
+    expect(queryAllByTestId('whenPrepared').length).toEqual(0);
```

Comment:
queryByTestId might be even clearer and shorter.
```suggestion
    expect(queryByTestId('whenPrepared')).toBeNull();
```